restservice/src/main/java/org/itevents/controller/EventRestController.java

```diff
@@ -26,8 +27,8 @@
     @Inject
     private UserService userService;
 
-    @RequestMapping(method = RequestMethod.GET, value = "/events/{eventID}")
-    public ResponseEntity<Event> getEventById(@PathVariable("eventID") int id) {
+    @RequestMapping(method = RequestMethod.GET, value = "/{eventId}")
+    public ResponseEntity<Event> getEventById(@PathVariable("eventId") int id) {
         Event event = eventService.getEvent(id);
         if (event == null) {
             return new ResponseEntity(HttpStatus.NOT_FOUND);
```

Comment:
https://www.dropbox.com/s/mgwh7bmvs1vauo7/%D0%91%D0%BB%D0%B8%D0%BD%D0%BE%D0%B2%20%D0%98.%D0%9D.%2C%20%D0%A0%D0%BE%D0%BC%D0%B0%D0%BD%D1%87%D0%B8%D0%BA%20%D0%92.%D0%A1.%20-%20Java.%20%D0%9C%D0%B5%D1%82%D0%BE%D0%B4%D1%8B%20%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F%20-%202013.pdf?dl=0